pkg/gateway/model_build_lattice_service.go

```diff
@@ -23,29 +24,39 @@ type LatticeServiceBuilder interface {
 	Build(ctx context.Context, httpRoute core.Route) (core.Stack, *latticemodel.Service, error)
 }
 
-type latticeServiceModelBuilder struct {
-	client.Client
+type LatticeServiceModelBuilder struct {
+	log         gwlog.Logger
+	client      client.Client
 	defaultTags map[string]string
 	Datastore   *latticestore.LatticeDataStore
-
-	cloud lattice_aws.Cloud
+	cloud       lattice_aws.Cloud
 }
 
-func NewLatticeServiceBuilder(client client.Client, datastore *latticestore.LatticeDataStore, cloud lattice_aws.Cloud) *latticeServiceModelBuilder {
-	return &latticeServiceModelBuilder{
-		Client:    client,
+func NewLatticeServiceBuilder(
+	log gwlog.Logger,
```

Comment:
thank you for propagating logger

Reply:
One general question, when we should use [this logger](https://github.com/aws/aws-application-networking-k8s/blob/d8b2e73263dff6356148ba4f46aa1dd35289aae7/cmd/aws-application-networking-k8s/main.go#L90), and when we should use `gwlog.FallbackLogge`r?

Reply:
fallback is a backup plan when it's hard to propagate logger and you need something for debugging. Rather than using glog.v6 it's better to use FallbackLogger. You can create an alias in package for convenience 
```
var log = gwlog.FallbackLogger

func foo() {
 - glog.V(6).Info("bla")
 + log.Debug("bla")
}
```